### DISCUSSION TITLE
How do I set topic permissions?

### DISCUSSION THREAD

**Discussion by ghost** [rabbitmq/messaging-topology-operator#174] [Q&A] [answered]
Hi!

I'm currently trying to configure a cluster using the messaging topology operator. I can't find if it's even possible to set the topic permissions. Is this feature hidden or hasn't it been added yet?

**Comment by ChunyiLyu** [comment, marked as answer]
@toda-boozt Sorry for the late response! No topic permission functionality hasn't been added to the topology operator yet.

**Reply by milank78git** [reply]
Hi,
is this feature in development? 
Thank you very much
M.
